doc/sphinx/src/using-eos.rst

```diff
@@ -216,6 +216,99 @@ available member function.
    eos.TemperatureFromDensityInternalEnergy(density.data(), energy.data(), temperature.data(),
                                             scratch.data(), density.size());
 
+The Evaluate Method
```

Comment:
Took a stab at documentation

Reply:
Thanks!

Reply:
Can you put a quick blurb about this up in the "The parallelism model" section? I think this would be good to advertise up there.

Reply:
Good point @jhp-lanl ! Done.